File: bookManageFrontDev/src/services/redisController.js
import request from '@/utils/request' 

/*
*@param: redis:class com.Httb.bookManage.mbg.entity.Redis
*/
export function saveRedis(redis) {
    return request({
        url: request.baseUrl + `/saveRedis`,
        method: 'POST',
        data:redis
    })
}
/*
*@param: rid:class java.lang.Integer

*/
export function selectRedisByResourcesId(rid) {
    return request({
        url: request.baseUrl + `/selectRedisByResourcesId`,
        method: 'GET',
        params:{
            rid:rid,

        }
    })
}

File: bookManageBackDev/src/main/java/com/Httb/bookManage/controller/RedisController.java
package com.Httb.bookManage.controller;

import com.Httb.bookManage.mbg.entity.Redis;
import com.Httb.bookManage.service.RedisService;
import org.springframework.web.bind.annotation.*;

import javax.annotation.Resource;

@RestController
public class RedisController {

    @Resource
    private RedisService redisService;

    /**
     * save redis配置
     */
    @PostMapping("/saveRedis")
    public Integer saveRedis(@RequestBody Redis redis) {

        return redisService.saveRedis(redis);
    }

    /**
     * 根据 资源id 查redis配置
     */
    @GetMapping("/selectRedisByResourcesId")
    public Redis selectRedisByResourcesId(@RequestParam Integer rid) {

        return redisService.selectRedisByResourcesId(rid);
    }
}
